runtimes/php/tutorial-laravel.md

```diff
@@ -98,3 +98,30 @@ Create a new environment variable called `CC_FS_BUCKET` and set `/storage/app:<b
 If you need to build your frontend assets (eg. javascript or CSS files), you can either add it as a step in your composer file, or you can add a post build hook with the `CC_POST_BUILD_HOOK` environment variable.
 
 For example, if you launch the build with `npm run prod`: `CC_POST_BUILD_HOOK=npm install && npm run prod`.
+
+### Optional: configure task scheduling
```

Comment:
No need to add "optional", the first sentence of the block states it.

Reply:
The point was to imitate the previous section's title. Also I believe this is more readable to state this in the title, so this can appear in the ToC. Should I remove it?

Reply:
Mh, okay, I didn't see the previous title using "optional" too, that's not something we usually do. Leave it for now, I'll open a separate issue for this as there are also problems with the titles in general for this document.